### DISCUSSION TITLE
cant build the latest master branch

### DISCUSSION THREAD

**Discussion by bluehock** [Auties00/WhatsappWeb4j#437] [General]
Hi, its really a fucking awesome project!  I love it so much, very cool! :)  
But  my IDEA says miss `ProtocolMessageBuilder` `ChatBuilder` class when build with git version `911e1bf` which is committed 2 days ago. could u please push the  build-able version?

**Comment by Auties00** [comment]
If you hit compile it will generate those classes automatically

**Reply by bluehock** [reply]
yes! you are right. but i got new error like `no instance(s) of type variable(s) U exist so that Optional<U> conforms to String`
![Uploading image.png…]()

**Comment by bluehock** [comment]
![QQ20240111-233457](https://github.com/Auties00/Cobalt/assets/49931659/cc05a9fc-d803-4a1f-8354-9d4475b8642a)

**Reply by Auties00** [reply]
My bad, I'll push the updated commit asap

**Reply by bluehock** [reply]
LOVE U
